benchmark/inference/utils.py

```diff
@@ -39,6 +39,13 @@ def get_dataset(name, root, use_sparse_tensor=False, bf16=False):
 
     data = dataset[0]
 
+    if name == 'ogbn-products':
+        split_idx = dataset.get_idx_split()
+        data.train_mask = split_idx['train']
```

Comment:
Technically not a mask but an index. Shall we use `index_to_mask` function to convert or ignore? :)